plugin/vim-printer.vim

```diff
@@ -22,6 +22,12 @@ let s:vim_printer_items_full = {
             \ 'sh': 'echo "{$}" "${$}"',
             \ 'bash': 'echo "{$}" "${$}"',
             \ 'zsh': 'echo "{$}" "${$}"'
+            \ 'java': 'System.out.println("{$}:" + {$});'
+            \ 'lua': 'print("{$}:", {$})'
```

Comment:
lua seems to add `\t` if you use `,` add a space in the first string and use append.